todo_app/app.py

```diff
@@ -1,11 +1,37 @@
-from flask import Flask
+from flask import Flask, render_template, request, redirect
+from todo_app.data.session_items import SessionItems
 
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
-
+si = SessionItems()
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items=si.get_items()
+    items = sorted(items, key=lambda x: x['status'], reverse=True)
+    return render_template("index.html", items=items)
+
+@app.route('/add-item', methods=["GET", "POST"])
+def add_item():
+    if request.method == "POST":
+        title = request.form.get("title", "")
+        si.add_item(title)
+        return redirect('/')
+    return render_template("add_item.html")
+
+@app.route('/change-status/<id>')
+def change_status(id):
```

Comment:
Often quicker to get this looking nice with this approach, but if we want to follow more semantic method-patterns, then it's a good habit to make this a POST endpoint, and wrap the button in a form rather than a link
(Ideally we might actually make it a PUT, and the below a DELETE, but using HTML forms limits us to just GET and POST 😞 )